File: foundation 3 month/lesson7/homework/ex_8.py
import random
def check(list1,list2):
	tot=list1+list2
	return tot

k=int(input("arr and ms  ni kiriting: "))
arr=list()
ms=list()
for x in range(k):
	arr.append(random.randint(10,50))
	ms.append(random.randint(10,50))
print(arr)
print(ms)
ans=check(arr,ms)
print(ans)

File: foundation/foundation 2 month/lesson6/homework/ex_8.c
#include <stdio.h>
#include <stdlib.h>
void check( char ms[])
{
	for(int i=0; ms[i] != '\0'; i++)
	{
		if(ms[i]>='0' && ms[i]<='9')
		{
		printf("%c",ms[i]);
		}
	}
}

int main()
{
	char str[100];
	printf("Kiriting: ");
	scanf("%[^\n]%*c",str);
	check(str);
}
